src/twitter_mood/mood_gatherer.py

```diff
@@ -1,15 +1,31 @@
-from .mood_interface import MoodInterface
+import collections
+
+from textblob import TextBlob
 
 
 class TwitterMoodGatherer:
 
-    def __init__(self, twitter_interface, sentiment_interface):
-        self.__twitter_interface = twitter_interface
-        self.__sentiment_interface = sentiment_interface
+    def __init__(self, twitter_api, query):
+        self.__twitter_api = twitter_api
+        self.__query = query
+        self.__sentiment_tool = TextBlob
```

Comment:
Is this necessary? I suppose it allows us to easily change our sentiment tool if we want to (or allow the user to change it).